src/django/api/views.py

```diff
@@ -425,6 +427,8 @@ class FacilitiesAutoSchema(AutoSchema):
     def get_link(self, path, method, base_url):
         if method == 'DELETE':
             return None
+        if path[-6:].lower() == 'merge/':
```

Comment:
Out of curiosity, would `if 'merge' in path` work? I used `if 'claim' in path` for suppressing the claim route, but I don't know whether there are cases when the path would be not in lowercase or if 'claim' (or 'merge') could appear validly in the path string prior to the end.

Reply:
`if 'merge' in path` is probably sufficient.

I added call to `lower()` instinctually because we are string comparing URLs which are case insensitive. However, because the value of `path` is not coming from a user, it will probably never be anything other than lowercase.

It's not likely that we will ever have another route under `/api/facilities` with `merge` in the path, but anchoring the match to the end of the path is a bit more explicit. That said, this would also match `/api/facilities/undo-merge`, so it is not actually as protective as it should be.

I'll switch to the simpler `if 'merge' in path` for consistency.

Reply:
That makes sense -- happy to use whichever idiom is best for suppressing `claim` also.

Reply:
Updated in FIXUP 9aedbdc